src/d3a/models/strategy/external_strategies/storage.py

```diff
@@ -24,7 +24,8 @@ def event_activate(self):
             f'{self.channel_prefix}/bid': self._bid,
             f'{self.channel_prefix}/delete_bid': self._delete_bid,
             f'{self.channel_prefix}/list_bids': self._list_bids,
-            f'{self.channel_prefix}/stats': self._area_stats
+            f'{self.channel_prefix}/stats': self._area_stats,
```

Comment:
As discussed in the @faizan2590 s PR. I think we deprecated this endpoint.

Reply:
Great, I'll delete it and the corresponding methods in the context of this PR.